jenkins/catapult/requirements.txt

```diff
@@ -1,2 +1 @@
 PyYAML==6.0.1
```

Comment:
don't you need conan package as dependency ?

Reply:
Yeah, I miss this.